src/configmanager.cpp

```diff
@@ -108,13 +108,15 @@ bool ConfigManager::load()
 		string[MYSQL_PASS] = getGlobalString(L, "mysqlPass", "");
 		string[MYSQL_DB] = getGlobalString(L, "mysqlDatabase", "forgottenserver");
 		string[MYSQL_SOCK] = getGlobalString(L, "mysqlSock", "");
+		string[HTTP_API_IP] = getGlobalString(L, "httpApiIp", "127.0.0.1");
```

Comment:
@ranisalt It depends on the Docker network setup. Also, if you have the proxy (e.g. nginx) in a different container, you want them to communicate directly, not exactly forwarding through the host just for that purpose.

Yes, let's not mak people recompile just because we're trying to make sure people don't commit suicide.